shell/packages/accounts-saml/saml_server.js

```diff
@@ -11,6 +11,10 @@ RoutePolicy.declare("/_saml/", "network");
 const HOSTNAME = Url.parse(process.env.ROOT_URL).hostname;
 
 Accounts.registerLoginHandler(function (loginRequest) {
+  if (!Accounts.identityServices.saml.isEnabled()) {
```

Comment:
Shouldn't this check go *after* the next one (which checks if this is a SAML login)?